File: Lv.2/최댓값과 최솟값.cpp
#include <string>
#include <vector>
#include <algorithm>
#include <functional>
#include <iostream>
using namespace std;

string solution(string s) {
    string answer = "";
    
    vector<int> arr;
    string temp = "";
    
    for(int i=0;i<s.length();i++){
        char cur = s[i];
        if(cur==' '){
            arr.push_back(stoi(temp));
            temp="";
            continue;
        }
        else{
            temp+=(cur);
        }
    }
    arr.push_back(stoi(temp));
    
    int min = *min_element(arr.begin(),arr.end());
    int max = *max_element(arr.begin(),arr.end());
    
    answer+=to_string(min);
    answer+=' ';
    answer+=to_string(max);
    
    return answer;
}

File: Java/LV2/최댓값과 최솟값.java
import java.util.*;

class Solution {
    public String solution(String s) {
        String answer = "";
        String temp = "";
        int counter = 0;
        ArrayList<Integer> al = new ArrayList<>();
        for(int i=0;i<s.length();i++){
            if(s.charAt(i)==' '){
                al.add(Integer.parseInt(temp));
                temp="";
                counter++;
                
            }
            else{
                temp+=s.charAt(i);
            }
        }
        al.add(Integer.parseInt(temp));
        
        Collections.sort(al,Collections.reverseOrder());
        String max = Integer.toString(al.get(0));
        String min = Integer.toString(al.get(counter));
        
        answer+=min;
        answer+=" ";
        answer+=max;
        return answer;
    }
}
